ext/json/tests/json_validate_requires.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+function json_validate_trycatchdump($json, $depth = 512, $flags = 0) {
+  try {
+    var_dump(json_validate($json, $depth, $flags));
+  } catch (JsonException $e) {
+    echo "JsonException: {$e->getCode()} {$e->getMessage()}". PHP_EOL;
+  } catch (Exception $e) {
+    echo "Exception: {$e->getCode()} {$e->getMessage()}". PHP_EOL;
+  } catch (Error $e) {
+    echo "Error: {$e->getCode()} {$e->getMessage()}". PHP_EOL;
+  }
+
+  var_dump(json_last_error(), json_last_error_msg());
```

Comment:
we are using 4 spaces in tests (at least the json ones).

Reply:
:heavy_check_mark: